versioned_docs/version-1.3.0/osx/01-get-started.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: ''
+sidebar_label: Get Started
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+import DevTools from '../../../src/pages/dev-tools.tsx';
+
+<DevTools />
```

Comment:
I only comment here
`devTools` naming seems a bit off, why we show devTools in get started.

i was expecting a structure like:
- getStarted
- - devToolds
- - ...etc

Reply:
Dev-tools was the name (and web route path) of the get started webpage. https://devs.aragon.org/dev-tools/
I can rename everything to `getStarted`